pages/linux/dnf.md

```diff
@@ -0,0 +1,24 @@
+# dnf
+
+>Package management utility for RHEL, Feodra, and CentOS (replaces yum)
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent dnf commands.
+
+`dnf update`
+
+- install a new package
```

Comment:
I think the first letter of a sentence should be consistently capitalized, so this line should become "Install a new package", and same for the two examples below.